theme/theme.scss

```diff
@@ -29,3 +29,14 @@
 [class*="ck-heading_heading"] {
 	font-weight: bold;
 }
+
+// Resize dropdown's button label.
+.ck-dropdown {
+	&.ck-heading-dropdown {
+		.ck-dropdown__button {
+			.ck-button__label {
+				width: 8em;
```

Comment:
Haven't thought about `Wählen Sie die Überschrift` :D. We need this style because button's label has default width. Maybe it should be changed to `min-width` - @oleq?.